### ISSUE TITLE
Fix tables in Module wiki

### ISSUE THREAD

**Issue by aornugent** [moja-global/FLINT#103]
**What does the current documentation state?**
The module wiki has tables that are not formatted correctly. 

**Where is this stated?**
https://github.com/moja-global/FLINT/wiki/3.4-Module-Science

**Why do you want to improve the statement?**
Fix the markdown syntax by adding extra spaces.

**Comment by shloka-gupta** [comment]
If I am not wrong, just markdown beautifier has to be applied to the document, right?

Can I go for it?

**Comment by aornugent** [comment]
Yes please!

**Comment by shloka-gupta** [comment]
So, I did the required changes,

but while creating a PR I am getting the message :
remote: Permission to moja-global/FLINT.wiki.git denied to chicken-biryani.
fatal: unable to access 'https://github.com/moja-global/FLINT.wiki.git/': The requested URL returned error: 403

**Comment by shloka-gupta** [comment]
These links say that :
"GitHub doesn't support pull requests for the wiki repository, only the main repository"
https://stackoverflow.com/questions/10642928/how-to-pull-request-a-wiki-page-on-github
https://docs.github.com/en/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages

**Comment by chicken-biryani** [comment]
https://docs.github.com/en/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages

**Comment by soumik2012** [comment]
@aornugent Can I try this ??

**Comment by shubhamkarande13** [comment]
@chicken-biryani 
I guess that it is not possible to create a PR for a wiki on GitHub. Could you please make a zip file of the wiki folder and upload it to Google Drive and share it here, so that @aornugent can update the wiki from his end? ✌

@aornugent it seems that only users with write access to the repo can push updates to the wiki. 😅

**Comment by mehabhalodiya** [comment]
I have visited this link: https://github.com/moja-global/FLINT/wiki/3.4-Module-Science

And seen the first table was having some issue of formatting.
I have tried to correct it as below. Let me know if it's alright so that I can complete further!

Preview:
![moja2](https://user-images.githubusercontent.com/73488906/128630066-753d7e18-3c4b-4353-b3ba-f6ba95017c2a.png)

@aornugent Have a look!

**Comment by shloka-gupta** [comment]
Hey, @mehabhalodiya thank you for doing this!! 😃🎉 

Could you put the changes of the wiki folder in a zip file and upload it to Google Drive and share the link over here, in the comment section, as stated by @shubhamkarande13 earlier?
(As Github doesn't support PRs for the wiki repository.)

**Comment by mehabhalodiya** [comment]
@chicken-biryani Yes I will put it up but first let me know that is it the way you want the format?

**Comment by shloka-gupta** [comment]
@mehabhalodiya
Yes. That is the correct format. However, I think there are a few more tables that require formatting.
Just apply a [markdown formatter](https://codebeautify.org/markdown-formatter) to the whole document and the work will get done!

**Comment by mehabhalodiya** [comment]
https://drive.google.com/file/d/1oUWyhBPpA3DKAzH90cxXknXJl2Ha0QeD/view?usp=sharing

@chicken-biryani @shubhamkarande13 @aornugent PTAL!

**Comment by apoorvthedude** [comment]
Hi, Please someone mail to @aornugent  to close the issue as the file is formatted by @mehabhalodiya.Now the wiki is in correct format.
She has shared the file.
